src/ui/components/search/searchcomponent.js

```diff
@@ -678,7 +678,7 @@ export default class SearchComponent extends Component {
       labelText: this.labelText,
       inputLabelIdName: this.inputLabelIdName,
       submitIcon: this.submitIcon,
-      searchBarContainerId: this._container.id,
+      searchBarContainerId: this._container.id || '',
```

Comment:
I think that `??` is preferable to `||` here because if the id is 0 and `||` us used, that will fall back to the empty string whereas `??` won't fall back if the value is 0